test/unittest/compiler/unittest_interpreter.cpp

```diff
@@ -34,9 +34,12 @@ makeGraph(const std::vector<LayerReprentation> &layer_reps) {
   auto graph = std::make_shared<nntrainer::GraphRepresentation>();
 
   for (const auto &layer_representation : layer_reps) {
-    std::shared_ptr<ml::train::Layer> layer = ac.createObject<ml::train::Layer>(
-      layer_representation.first, layer_representation.second);
-    graph->addLayer(std::static_pointer_cast<nntrainer::Layer>(layer));
+    std::shared_ptr<nntrainer::Layer> nntr_layer =
+      ac.createObject<nntrainer::Layer>(layer_representation.first,
+                                        layer_representation.second);
+    std::shared_ptr<nntrainer::LayerNode> layer =
+      std::make_unique<nntrainer::LayerNode>(nntr_layer);
```

Comment:
I guess nntr_layer can better be moved in the near future?

Reply:
yes, will update this.